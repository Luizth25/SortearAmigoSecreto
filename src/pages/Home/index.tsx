import Card from "components/Card";
import Formulario from "components/Formulario";
import Header from "components/Header";
import ListaParticipantes from "components/ListaParticipantes/index.tsx";
import RodaPe from "components/RodaPe";

const Home = () => {
  return (
    <>
      <Header />
      <Card>
        <section>
          <h2>Vamos come√ßar</h2>
          <Formulario />
          <ListaParticipantes />
          <RodaPe />
        </section>
      </Card>
    </>
  );
};

export default Home;
