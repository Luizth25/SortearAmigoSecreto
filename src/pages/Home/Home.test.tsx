import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";

import Home from "pages/Home";

const mockDeNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockDeNavegacao,
  };
});

describe("a pagina de configuração", () => {
  test("deve ser renderizida corretamente", () => {
    const { container } = render(
      <RecoilRoot>
        <Home />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
