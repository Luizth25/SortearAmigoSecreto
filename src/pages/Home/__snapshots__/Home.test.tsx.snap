// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuração deve ser renderizida corretamente 1`] = `
<div>
  <header
    class="sc-jSUZER cFerTL"
  >
    <div>
      <img
        alt="Logo"
        src="logo.png"
      />
    </div>
    <div
      class="sc-gKPRtg iTxCIu"
    >
      <img
        alt="Participante"
        src="participante.png"
      />
    </div>
  </header>
  <div
    class="sc-bcXHqe hzCVtx"
  >
    <section>
      <h2>
        Vamos começar
      </h2>
      <form>
        <div
          class="sc-gswNZR eVwNIr"
        >
          <input
            class="sc-dkrFOg fSMxc"
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            class="sc-hLBbgP gJyAZy"
            disabled=""
          >
            Adicionar
          </button>
        </div>
      </form>
      <ul />
      <footer
        class="sc-iBYQkv jtYJUD"
      >
        <button
          class="sc-ftTHYK bcusvr"
          disabled=""
        >
          Iniciar Brincadeira
        </button>
        <img
          alt="sacolas de papel"
          src="sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
